and-bible/app/src/main/java/net/bible/android/control/bookmark/BookmarkControl.java

```diff
@@ -125,12 +130,44 @@ public String getBookmarkVerseKey(BookmarkDto bookmark) {
 		return keyText;
 	}
 
+	public String getBookmarkBookUsed(BookmarkDto bookmark) {
+		String bookUsed = "";
+		try {
+			bookUsed = bookmark.getBookUsed();
+		} catch (Exception e) {
+			Log.e(TAG, "Error getting book used", e);
+		}
+		return bookUsed;
+	}
+
 	public String getBookmarkVerseText(BookmarkDto bookmark) {
 		String verseText = "";
 		try {
 			CurrentBiblePage currentBible = activeWindowPageManagerProvider.getActiveWindowPageManager().getCurrentBible();
 			Versification versification = currentBible.getVersification();
 			verseText = swordContentFacade.getPlainText(currentBible.getCurrentDocument(), bookmark.getVerseRange(versification));
+
+			String bookUsed = bookmark.getBookUsed();
+			if ((bookUsed != null) && (!bookUsed.isEmpty()) && !bookUsed.equals(getCurrentBookUsed())) {
+				//Book book = currentPageControl.getCurrentBible().getSwordDocumentFacade().getDocumentByInitials(abbrev);
+				Book book = currentBible.getSwordDocumentFacade().getDocumentByInitials(bookUsed);
+				if (book != null) {
+					verseText = swordContentFacade.getPlainText(book, bookmark.getConvertibleVerseRange().getVerseRange(((AbstractPassageBook) book).getVersification()));
+					// Below copied can be adapted to cater for font
+//					if (verseText.length() > 0) {
+//
+//						// does this book require a custom font to display it
+//						File fontFile = null;
+//						String fontForBook = fontControl.getFontForBook(book);
+//						if (StringUtils.isNotEmpty(fontForBook)) {
+//							fontFile = fontControl.getFontFile(fontForBook);
+//						}
+//
+//						// create DTO with all required info to display this Translation text
+//						retval.add(new TranslationDto(book, text, fontFile));
+//					}

```

Comment:
Some books may need a special font to display. For now I do not cater for this in the bookmark list - not sure that it is necessary. Your input on this will be welcomed.

I copied the corresponding code from CompareTranslationsControl (where the font is supported), but commented it out with a explanatory comment at the start. This can be taken out/modified depending on the way we want to go.